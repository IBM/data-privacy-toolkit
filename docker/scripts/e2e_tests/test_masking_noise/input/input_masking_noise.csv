M,22
F,23
M,27
