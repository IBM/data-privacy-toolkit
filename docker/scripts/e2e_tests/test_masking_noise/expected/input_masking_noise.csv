M,20.120407781579843
F,22.617264786128636
M,26.559843266589116